src/quo2/foundations/colors.cljs

```diff
@@ -183,6 +183,23 @@
       (let [base-color (get-in colors-map [(keyword color) suffix])]
         (if opacity (alpha base-color (/ opacity 100)) base-color))))))
 
+(defn map-opacity-to-hex [opacity]
```

Comment:
It's more conventional in Clojure to name such functions with an arrow indicating the transformation direction. So instead of `map-opacity-to-hex`, use `opacity->hex`.

Reply:
Thanks for the new info